Makefile

```diff
@@ -93,8 +93,8 @@ endif
 all: all-staged
 
 all-unstaged: compile-gcc-compatible compile-msvc-compatible \
-  compile-portable-gcc-compatible \
-  dist/wasm/package.json obj/libhaclml.cmxa
+  compile-portable-gcc-compatible compile-wasm \
```

Comment:
this is going to call gcc, isn't it? see rule for `compile-%`

why not as before and take a dependency on dist/wasm/Makefile.basic, to force generation, but not compile in C?

Reply:
Thanks for the note, I replaced it with `dist/wasm/Makefile.basic`